src/actor/Verifier.spec.ts

```diff
@@ -142,13 +147,40 @@ describe('Verifier', () => {
       })
       .finalize(true, verifier, claimer.getPublicIdentity())
     expect(session).toBeDefined()
+    expect(session.requestedProperties[0].properties).not.toContain(
+      'legitimation'
+    )
+    expect(session.requestedProperties[0].properties).not.toContain(
+      'delegation'
+    )
     expect(request.body.type).toEqual(MessageBodyType.REQUEST_CLAIMS_FOR_CTYPES)
     if (request.body.type === MessageBodyType.REQUEST_CLAIMS_FOR_CTYPES) {
       expect(request.body.content.allowPE).toBeTruthy()
       expect(request.body.content.peRequest).toBeDefined()
       expect(request.body.content.ctypes).toEqual(['this is a ctype hash'])
     }
   })
+  it('request privacy enhanced presentation w/ legitimations and delegation', async () => {
+    const { session, message: request } = await Verifier.newRequestBuilder()
+      .requestPresentationForCtype({
+        properties: ['name', 'and', 'other', 'attributes'],
+        legitimations: true,
+        delegation: true,
+      })
+      .finalize(true, verifier, claimer.getPublicIdentity())
+    expect(session).toBeDefined()
+    expect(session.requestedProperties[0].properties).not.toContain(
+      'claim.cTypeHash'
+    )
+    expect(session.requestedProperties[0].properties).toContain('delegationId')
+    expect(session.requestedProperties[0].properties).toContain('delegationId')
```

Comment:
one of them was supposed to be the legitimations, right?

Reply:
Fixed in 5db18c9